### ISSUE TITLE
Error importing run_sf_prediction

### ISSUE THREAD

**Issue by pcamach2** [MICA-MNI/micaopen#5]
I am interested in testing the model from the recent publication on prediction of functional connectivity from the structural connectome. I set up a Singularity image based on the Python 3.8 image with the dependencies listed in this repo, but I run into this error when trying to import the function [as suggested](https://github.com/MICA-MNI/micaopen/tree/master/sf_prediction#code):

```
>>> from utils import run_sf_prediction
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/micaopen/sf_prediction/utils.py", line 10, in <module>
    from pymanopt.manifolds import Rotations
ImportError: cannot import name 'Rotations' from 'pymanopt.manifolds' (/usr/local/lib/python3.8/site-packages/pymanopt/manifolds/__init__.py)
```

Is there a specific version of pymanopt that is required to run this code? 
Any help would be greatly appreciated!

**Comment by pcamach2** [comment]
The import error resolves if I pin the version of pymanopt to 0.2.5

**Comment by OualidBenkarim** [comment]
Yes, I think `Rotations` is `SpecialOrthogonalGroup` in current version, but there are many other breaking changes. We will stick to 0.2.5 for now

Thanks @pcamach2
